platform/security/common/pom.xml

```diff
@@ -176,17 +176,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.27</minimum>
+                                            <minimum>0.40</minimum>
```

Comment:
🎉

Reply:
I think I figured out what happened. Some unit tests had errors, which reduced coverage. However, the coverage failure was the last thing in the console output, so it is the "bug" I fixed first. After fixing the unit tests, the coverage went back up.